### PR TITLE
DTGB-276: Fixed breadcrumbs padding issue in styleguide and gent_base.

### PR DESCRIPTION
**Pull request by meirege**
## Description
Fixed whitespace issue + JS issue in gent_base. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [x] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/31-molecules/building-blocks/breadcrumbs/breadcrumbs.twig [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
-  <h2 id="system-breadcrumb" class="visually-hidden">{{ 'Breadcrumb' }}</h2>
-  <ol class="content-container breadcrumb">
-    {% for item in items %}
-      <li>{{ item }}</li>
-    {% endfor %}
-  </ol>
+  <div class="content-container">
+    <h2 id="system-breadcrumb" class="visually-hidden">{{ 'Breadcrumb' }}</h2>
+    <ol class="breadcrumb no-style">
+      {% for item in items %}
```
Comment (delrueba) [inline styleguide/components/31-molecules/building-blocks/breadcrumbs/breadcrumbs.twig]: Suggest adding [{% spaceless %}](https://twig.symfony.com/doc/2.x/tags/spaceless.html)
See: https://codepen.io/chriscoyier/pen/hmlqF

File: templates/navigation/breadcrumb.html.twig [hunk 2]
```diff
@@ -8,18 +8,20 @@
...
-        {% endif %}
-      </li>
-    {% endfor %}
-    </ol>
+  <nav class="breadcrumb section--default" role="navigation" aria-labelledby="system-breadcrumb">
+    <div class="content-container">
+      <h2 id="system-breadcrumb" class="visually-hidden">{{ 'Breadcrumb'|t }}</h2>
+      <ol class="breadcrumb no-style">
+        {% for item in breadcrumb %}
+          <li>
```
Comment (delrueba) [inline templates/navigation/breadcrumb.html.twig]: Suggest adding {% spaceless %}

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/building-blocks/breadcrumbs/_breadcrumbs.scss [hunk 3]
```diff
@@ -1,51 +1,49 @@
...
+      a {
+        padding: .4rem;
+        border-radius: $radius-1;
+        text-decoration: none;
 
-          &:hover {
-            background-color: $bg-hover-breadcrumb;
-          }
+        &:hover {
+          background-color: $bg-hover-breadcrumb;
```
Comment (delrueba) [inline styleguide/components/31-molecules/building-blocks/breadcrumbs/_breadcrumbs.scss]: There's a mixin to include :focus

File: templates/layout/page.html.twig [hunk 4]
```diff
@@ -59,7 +59,7 @@
     <header class="section--wrapper header" role="banner">
       <div class="content-container">
         {% if page.nav_mobile %}
-          <nav class="hamburger-menu visible-mobile header-component">
+          <nav class="hamburger-menu visible-mobile header-compone .5rem -.35renfinmnt">
```
Comment (delrueba) [inline templates/layout/page.html.twig]: Typo?

**Review by delrueba — CHANGES_REQUESTED** [review]

File: templates/navigation/breadcrumb.html.twig (hunk 2 above)
Comment (meirege) [reply templates/navigation/breadcrumb.html.twig]: Agreed. Added.

File: templates/layout/page.html.twig (hunk 4 above)
Comment (meirege) [reply templates/layout/page.html.twig]: Fixed

File: styleguide/components/31-molecules/building-blocks/breadcrumbs/breadcrumbs.twig (hunk 1 above)
Comment (meirege) [reply styleguide/components/31-molecules/building-blocks/breadcrumbs/breadcrumbs.twig]: Agreed. Added.

File: styleguide/components/31-molecules/building-blocks/breadcrumbs/_breadcrumbs.scss (hunk 3 above)
Comment (meirege) [reply styleguide/components/31-molecules/building-blocks/breadcrumbs/_breadcrumbs.scss]: Used the element-status() mixin as suggested.

**Review by delrueba — APPROVED** [review]

**Pull request merged**